src/warc2zim/templates/load.js

```diff
@@ -6,15 +6,27 @@ async function main() {
 
   var worker = new Worker("./sw.js");
 
-  const parts = window.location.href.split("/");
-  const inx = parts.indexOf("A");
-  const name = parts[inx - 1];
+  var prefix = window.location.href.slice(0, window.location.href.indexOf("/A/"));
```

Comment:
That won't work on kiwix-serve if it's being used with a `--urlRootLocation` that contains a `/A/`.
Opening a separate issue for this as this might not be trivial to fix.

Reply:
Added better detection, now using regex to find a `/A/` before a string with `.`, so `http://example.com/A/path/A/example.com` will find `/A/example.com/` not `/A/path/A/example.com`

Of course, can't be entirely full proof.. if someone has `http://example.com/A/example.com/A/example.com/`
then this is ambiguous and not much we can do w/o specifying the prefix explicitly, and I hope that doesn't happen!